jfx_bridge/bridge.py

```diff
@@ -367,7 +368,21 @@ def to_dict(self):
             _type = type(self.local_obj).typeName
         except AttributeError:
             _type = type(self.local_obj).__name__
-        return {HANDLE: self.handle, TYPE: _type, ATTRS: self.attrs, REPR: repr(self.local_obj)}
+
+        if hasattr(self.local_obj, "typeName"):
+            _name = self.local_obj.typeName
```

Comment:
Why add _name, what do you get that's different from _type?

Reply:
For a Class like `ghidra.GhidraApplicationLayout` `_type` will be something like `JavaClass` and __name__ is `GhidraApplicationLayout`. But I need the `ghidra.GhidraApplicationLayout` which is in `.typeName` for a Class object